packages/ui/src/helpers/storage/file-uploader/utils/uploader.tsx

```diff
@@ -0,0 +1,31 @@
+import { FileName } from '../../../../types/storage';
```

Comment:
this file should have `.ts` extension, not `.tsx`.

Reply:
Done